src/main/java/me/coley/recaf/parse/bytecode/Disassembler.java

```diff
@@ -307,7 +314,8 @@ private void visitLdcInsn(StringBuilder line, LdcInsnNode insn) {
 		line.append(' ');
 		if(insn.cst instanceof String) {
 			String str = insn.cst.toString();
-			str = EscapeUtil.escapeCommon(str);
+			// TODO: support for escaping "
+			str = EscapeUtil.escape(str);
```

Comment:
Because our parser is a hacky pile of substringing mess, I don't think we actually need to escape `"` for now.